### DISCUSSION TITLE
AWS Cloud formation stack failing

### DISCUSSION THREAD

**Discussion by hect1c** [sorry-cypress/sorry-cypress#703] [Q&A] [answered]
Hello,

I am trying to setup sorry-cypress self hosting on AWS. I click on the AWS cloud formation stack and start the creation of the stack it eventually fails to create any and all resources and then rolls back. 

I've tried using different regions, roles, full priv roles, same thing always happens and it provides not details as to why, is there any way for me to find out why this is happening and how I can get up and running quickly. Not sure what information to provide.

For each resource it tries to create I get a
<img width="1219" alt="image" src="https://user-images.githubusercontent.com/1482519/205290981-02e27754-551c-4e91-b0ad-269dc67b732a.png">

Thanks ahead of time for any feedback.

**Comment by dayaneCsantos** [comment]
@hect1c Hello! I'm having the same error, can you please share how you resolved it?

**Reply by hect1c** [reply]
Hey @dayaneCsantos I didn't. I moved to their cloud service [https://currents.dev](https://currents.dev) instead of self hosting but this obviously incurs a cost. It was something worth it for us as we didn't have the capacity to look into this further and needed it up and running ASAP. Sorry I couldn't help further

**Reply by dayaneCsantos** [reply, marked as answer]
Hey @hect1c, thank you very much. As an alternative I used the command via AWS CLI and it worked!
